facade/src/main/java/org/jboss/pnc/facade/rsql/mapper/AbstractRSQLMapper.java

```diff
@@ -65,6 +68,13 @@ public AbstractRSQLMapper(Class<DB> type) {
                 return mapEntity(from, toEntity(name), selector.next());
             }
         }
+        if (toEntitySet(name) != null) {
+            if (selector.isFinal()) {
+                return (PluralAttributePath<?>) from.get(toEntitySet(name));
```

Comment:
Why are you casting it to `PluralAttributePath` when the return type is only `Path`?

Reply:
I wanted it to be more specific to make it clear that its for multiple attributes, and from.get returns Expression so some cast is need. You think that casting to Path<> would be better?

Reply:
yes, this is unnecessarily too specific.